website/blog/2019-09-07-managing-multiples-projects-with-lerna-and-yarn-workspaces.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Managing multiples projects with Lerna and Yarn Workspaces
+author: Sergio Herrera
+authorURL: https://github.com/sergiohgz
+authorImageURL: https://avatars3.githubusercontent.com/u/14012309
+authorTwitter: sergiohgz
+---
+
+Verdaccio is a project with a big ecosystem, composed by several projects, each one with its own configurations and ways to manage. This fact transforms a simple maintenance operation, like updating a common dependency, into a real hard work.
+
+We saw a problem, these configurations entropry made harder to work with all the projects. So, we needed to simplify and unify them to make it easier. We need a **monorepo**.
+
+A monorepo is a project that can manage a collection of subprojects in a simple and unified way. There are many monorepos that Javascript developers are using daily, like [Babel](https://babeljs.io/), [Create React App](https://create-react-app.dev/) or [Material UI](https://material-ui.com/).
+
+Now, we are proud to announce our [monorepo](https://github.com/verdaccio/monorepo), our big ecosystem joined in only one repository. This is the first part of a series of articles where we will try to explain our motivation, what you need to setup your own, how to improve their management, how to work with and how to improve some workflows (CI, code quality, etc).
```

Comment:
> Now, we are proud to announce our [monorepo](https://github.com/verdaccio/monorepo), our big ecosystem joined in only one repository. 

(here we would need a sentence that actually describe how we solved our issue).

> This is the first part of a series of articles where we will try to explain our motivation, what you need to setup your own, how to improve their management, how to work with and how to improve some workflows (CI, code quality, etc).

This article is tje first part of a series of articles where we will try to explain our motivation about to a setup by your own, improve the management and workflows (CI, code quality, etc).

Reply:
For the first part, what do you think about `Now, we are proud to announce our monorepo, our big ecosystem joined in only one repository using Lerna and Yarn Workspaces.` I think the article itself will introduce how we solve it